packages/schematics/angular/application/files/.browserslistrc.template

```diff
@@ -5,8 +5,11 @@
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
-last 2 versions
-Firefox ESR
-not dead
-not IE 9-11 # For IE 9-11 support, remove 'not'.
\ No newline at end of file
+last 1 Chrome version
```

Comment:
Can you add a link to the AIO browser support page to the file as well?

Reply:
Thanks a good Idea!